content/api-reference/event-type-store/event-type-store.md

```diff
@@ -0,0 +1,7 @@
+---
+uid: EventTypeStore
+---
+
+# Event Type Store
+
+The Event Type Store APIs allow you to create, read, update, and delete types, which are then converted to a GraphQL schema.
```

Comment:
"Event Type Store" isn't capitalized in event-data-overview.md

Should be consistent either way, I think.